sklearn/feature_selection/tests/test_feature_select.py

```diff
@@ -402,36 +405,44 @@ def test_select_fdr_regression():
     Test that fdr heuristic actually has low FDR.
     """
     def single_fdr(alpha, n_informative, random_state):
-        n_features = 5 * n_informative
-
-        X, y = make_regression(n_samples=150, n_features=n_features,
+        X, y = make_regression(n_samples=150, n_features=20,
                                n_informative=n_informative, shuffle=False,
-                               random_state=random_state, noise=10)
+                               random_state=random_state, noise=5)
```

Comment:
I changed that while I was trying to debug. It does not change the outcome of the test.